src/index.js

```diff
@@ -92,7 +97,9 @@ export default class Routes extends Component {
                 render={() => (
                   <Request url={`/api/v1/instructors/${user.instructor_id}`}>
```

Comment:
this full url is on the `user` object that is returned by calling the `user_url` on the jwt.